src/ripple/protocol/impl/TxFormats.cpp

```diff
@@ -55,6 +55,8 @@ TxFormats::TxFormats()
             {sfClearFlag, soeOPTIONAL},
             {sfTickSize, soeOPTIONAL},
             {sfTicketSequence, soeOPTIONAL},
+            {sfNotValidBefore, soeOPTIONAL},
+            {sfNotValidAfter, soeOPTIONAL},
```

Comment:
The new fields `sfNotValidBefore` and `sfNotValidAfter` are comparable to the existing field `sfLastLedgerSequence` inside `commonFields`. So could the new fields be promoted to `commonFields` since they are also `soeOPTIONAL`?